verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -482,9 +483,20 @@ class LinelistTable extends React.Component<Props, LinelistTableState> {
     }
 
     downloadCases(): void {
+        let downloadUrl = '/api/cases/download';
+        const trimmedQ = this.state.search.trim();
+        if (trimmedQ) {
+            downloadUrl += '?q=' + encodeURIComponent(trimmedQ);
+        }
+        this.setState({ error: '' });
         axios
-            .get('/api/cases/download')
-            .then((response) => fileDownload(response.data, 'cases.csv'));
+            .get(downloadUrl)
```

Comment:
I would expect the get() method from axios to do the URI encoding no? do we really have to do it ourselves?

Reply:
Yes we have to do it ourselves :( throws an error if we don't. We had to do it in the request for table data as well.